### PR TITLE
Blank page on Dashboard

### PR DESCRIPTION
**Pull request by FullStackTiger**
**What does this pull request do? Explain your changes. (required)**

- When it runs through the flow on Livepeer.monster, it gets a blank page after loading. 

**Specific updates (required)**

- Added email verification function to every page on frontend side
- Remove allowUnverified param function on the auth middleware

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by adamsoffer — CHANGES_REQUESTED** [review]
Instead of including this `emailVerificationMode` conditional on every page, I think we should only include once in a higher order component. Cleaner.

**Comment by adamsoffer** [comment]
@FullStackTiger  hmm on second thought, possible to just add one conditional to the layout component rather than every page? This solution still requires we wrap each page with the HOC which would be good to avoid if possible.

**Review by adamsoffer — APPROVED** [review]
LGTM. @iameli want to give this a quick look over as well since it touches the API?

**Comment by adamsoffer** [comment]
@FullStackTiger I just noticed it's failing a prettier check.

**Comment by FullStackTiger** [comment]
> @FullStackTiger I just noticed it's failing a prettier check.

I fixed this prettier issue with another PR, already merged it to the master

File: packages/www/pages/app/admin/keys.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { getTabs } from "../admin";
 const Keys = () => {
   useLoggedIn();
   const { user, logout } = useApi();
-  if (!user || user.emailValid === false) {
+  if (!user) {
```
Comment (iameli) [inline packages/www/pages/app/admin/keys.tsx:13]: Example of the parameter I was discussing:

```suggestion
  if (!user || user.emailValid === req.config.requireEmailVerification) {
```

**Review by iameli — CHANGES_REQUESTED** [review]
Let's do the same thing we did on the front-end and add a new configuration parameter in `src/parse-cli.js` — `--require-email-verification`. It can default to false.

I left an example on one of the routes for what a check could look like. I'd also be okay with adding a new middleware helper that implements that check so we don't have to copy the `req.config.requireEmailVerification` everywhere.

File: packages/www/pages/app/admin/keys.tsx (hunk 1 above)
Comment (victorges) [reply packages/www/pages/app/admin/keys.tsx:13]: Maybe we need to check both separately? Not sure what we want to happen on a `emailValid = true` and `req.config.requireEmailVerification = false` case.

```suggestion
  if (!user || (!user.emailValid && req.config.requireEmailVerification)) {
```

**Review by pglowacky — APPROVED** [review]

**Pull request merged**